### ISSUE TITLE
wp create post with --meta_input does not seem to work as expected.

### ISSUE THREAD

**Issue by Mr-Alexander** [wp-cli/entity-command#202]
When I try the following command:

```
wp post create --post_title='A post' --post_content='Just a small post.' --meta_input='{"key1":"value1","key2":"value2"}'
```

I get the following response:

`Warning: Invalid argument supplied for foreach() in /wp-includes/post.php on line 3364`.

Also, in your documentation, the `'` is missing from the end of the command

`meta_input='{"key1":"value1","key2":"value2"}'`

**Comment by wojsmol** [comment]
for documentation part I created PR https://github.com/wp-cli/entity-command/pull/203

**Comment by danielbachhuber** [comment]
Fixed with #204

**Comment by izakman** [comment]
Hi, I am getting the same response when I try to add meta with wp post create:

`wp post create --post_title="Tester" --post_type="post" --meta_input='{"testkey":"test"}'`

`Warning: Invalid argument supplied for foreach() in C:\xxxxxxx\wp-includes\post.php on line 3527`
